components/review-sections/HouseholdReviewSection.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import useReviewSectionRows from "../../helpers/useReviewSectionRows";
+import householdSteps from "../../steps/household";
+import Storage from "../../storage/Storage";
+import { ReviewSection } from "../ReviewSection";
+
+export const HouseholdReviewSection: React.FunctionComponent = () => {
+  return (
+    <ReviewSection
+      section={{
+        heading: "Household",
+        rows: [...useReviewSectionRows(Storage.ProcessContext, householdSteps)],
```

Comment:
```suggestion
        rows: useReviewSectionRows(Storage.ProcessContext, householdSteps),
```

No need to destructure here.

Reply:
Same for the other sections.